README.md

```diff
@@ -312,19 +312,19 @@ For simpler use cases where session management isn't needed:
 const app = express();
 app.use(express.json());
 
-const transport: StreamableHTTPServerTransport = new StreamableHTTPServerTransport({
-  sessionIdGenerator: undefined, // set to undefined for stateless servers
-});
-
-// Setup routes for the server
-const setupServer = async () => {
-  await server.connect(transport);
-};
-
 app.post('/mcp', async (req: Request, res: Response) => {
-  console.log('Received MCP request:', req.body);
   try {
-      await transport.handleRequest(req, res, req.body);
+    const server = getServer(); 
+    const transport: StreamableHTTPServerTransport = new StreamableHTTPServerTransport({
```

Comment:
Maybe add comments explaining why we're instantiating a new transport each time?